docs/pages/router/advanced/native-tabs.mdx

```diff
@@ -11,9 +11,9 @@ import { FileTree } from '~/ui/components/FileTree';
 
 > **important** Native tabs is an experimental feature available in SDK 54 and above, and its API is subject to change.
```

Comment:
Should we add a video to native tabs as well?

https://www.youtube.com/watch?v=QqNZXdGFl44

Not sure if this is just a short video for now, or intended as part of docs

Reply:
Video link added:

![Uploading CleanShot 2025-10-07 at 18.25.09@2x.png…]()